server/views/formPages/incident/newIncident.html

```diff
@@ -51,7 +51,7 @@ <h1 class="govuk-heading-xl mainHeading">New use of force incident</h1>
         name: "forceType",
         fieldset: {
           legend: {
-            text: "Was the use of force plannned?",
+            text: "Was the use of force planned?",
             isPageHeading: false,
```

Comment:
😆

Reply:
That's been in for a such a long time and noooone noticed!